<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>720</width>
    <height>512</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QRadioButton" name="emoneyButton_3">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>190</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Shopy</string>
   </property>
  </widget>
  <widget class="QPushButton" name="confirmButton">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>440</y>
     <width>540</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 255, 133);</string>
   </property>
   <property name="text">
    <string>Confirm</string>
   </property>
  </widget>
  <widget class="QLabel" name="background">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>720</width>
     <height>512</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../images/metodePembayaranPic.png</pixmap>
   </property>
  </widget>
  <widget class="QRadioButton" name="cardButton_1">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>340</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="text">
    <string>PayPay</string>
   </property>
  </widget>
  <widget class="QLabel" name="metodePembayaran">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>20</y>
     <width>500</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Open Sans</family>
     <pointsize>-1</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font-family: 'Open Sans';font-size: 48px;color: #FFFFFF;</string>
   </property>
   <property name="text">
    <string>Metode Pembayaran</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="emoneyButton_1">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>130</y>
     <width>80</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="text">
    <string>AVA</string>
   </property>
  </widget>
  <widget class="QLabel" name="emoney">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>100</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>E-Money</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="cardButton_2">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>370</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Main Card</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="codButton">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>400</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Cash on Delivery</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="emoneyButton_2">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>160</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="text">
    <string>GoodPay</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="bankingButton_2">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>280</y>
     <width>80</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="text">
    <string>ABC</string>
   </property>
  </widget>
  <widget class="QLabel" name="card">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>310</y>
     <width>221</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Credit / Debit Card</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="bankingButton_1">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>250</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Sejahtera</string>
   </property>
  </widget>
  <widget class="QLabel" name="onlineBanking">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>220</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Online Banking</string>
   </property>
  </widget>
  <zorder>background</zorder>
  <zorder>emoneyButton_3</zorder>
  <zorder>confirmButton</zorder>
  <zorder>cardButton_1</zorder>
  <zorder>metodePembayaran</zorder>
  <zorder>emoneyButton_1</zorder>
  <zorder>emoney</zorder>
  <zorder>cardButton_2</zorder>
  <zorder>codButton</zorder>
  <zorder>emoneyButton_2</zorder>
  <zorder>bankingButton_2</zorder>
  <zorder>card</zorder>
  <zorder>bankingButton_1</zorder>
  <zorder>onlineBanking</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
